lib/ngx/ssl.lua

```diff
@@ -164,6 +167,25 @@ function _M.server_name()
     return nil, ffi_str(errmsg[0])
 end
 
+function _M.client_addr()
```

Comment:
Need some test cases in the existing test suite to cover this new Lua API function.